### ISSUE TITLE
[Usability] : Location of game control in mobile version

### ISSUE THREAD

**Issue by aniljava** [neagle/gokibitz#141]
The mobile version has the game control (forward, backward) on top. I see following problems with this design and recommend it to be placed down underneath the board.

1. Difficult single hand navigation on large screen phones.
2. Palm/Thumb area covers the board while navigating thereby taking away the instant view of move.

**Comment by aniljava** [comment]
More info: Found different behavior on different sizes. But mostly on standard mobile screen the controls are at top.

![1483490092](https://cloud.githubusercontent.com/assets/412569/21628224/0232372c-d1db-11e6-8a2e-b6d54f3facc6.jpg)
![1483490106](https://cloud.githubusercontent.com/assets/412569/21628225/0241c610-d1db-11e6-8dfd-26ab8bf82ca4.jpg)

**Comment by neagle** [comment]
Yeah, I actually _used_ to have the bar below the game in the narrowest mobile view, but then I moved it based on a friend's feedback that he couldn't scroll from the game down to the comments without accidentally hitting the button to move.

There are problems with either placement. I'd really like to come up with a few more significant innovations for solving a number of design issues with the mobile view.
